src/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -185,13 +185,18 @@ public void setRiskConfidence(int risk, int confidence) {
 		this.risk = risk;
 		this.confidence = confidence;
 	}
-	
+	/**
+	 * @deprecated (2.4.4) Replaced by {@link #setName}.

```

Comment:
Next version should be 2.5.0 (because of the enhancements and deprecations).
I usually add `TODO add version`, that can be later replaced when in the release branch (but the former should be ok too).